templates/example_cell_clustering.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Like in pixel clustering, I think it would help to have max_k and cap as parameters here, as people will likely want to change those. Also add prefix to all the files.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p><code>cell_cluster_name</code>  and <code>cell_consensus_name</code>  already have the prefix. I can add <code>max_k</code>  and <code>cap</code>.</p>